projects/draco.cmake

```diff
@@ -10,6 +10,3 @@ superbuild_add_project(draco
     -DBUILD_SHARED_LIBS=ON
     -DCMAKE_MACOSX_RPATH=OFF
```

Comment:
Dont you need to force PIC ? Do you need to set the install dir ?

Reply:
PIC is not necessary because draco is built as a shared lib in the SB.  
The install prefix is handled by the SB automatically and seems to work fine